src/Tribe/Customizer/General_Theme.php

```diff
@@ -25,12 +25,69 @@ public static function instance() {
 	}
 
 	/**
+	 * Gets the Colors for the Button background
+	 *
+	 * @since  F17.5
+	 *
+	 * @param  array  $settings Section array of settings
+	 * @return string
+	 */
+	protected function get_button_bg_color( $settings = array() ) {
+		$scheme = $this->sanitize_featured_color_choice( $settings['featured_color_scheme'] );
+		$schemes = $this->get_featured_color_schemes();
+
+		if ( 'custom' === $scheme ) {
+			$button_bg = $settings['featured_color_scheme_custom'];
+		} else {
+			$button_bg = $schemes[ $scheme ]['colors'][0];
+		}
+
+		if ( ! $button_bg ) {
+			$button_bg = $schemes['default']['colors'][0];
+		}
+
+		return $button_bg;
+	}
+
+	/**
+	 * Creates the Section ghost settings for Customizer
+	 *
+	 * @since  F17.5
+	 *
+	 * @param  array  $settings Section array of settings
+	 * @return array
+	 */
+	public function create_ghost_settings( $settings = array() ) {
+		if ( ! empty( $settings['featured_color_scheme'] ) ) {
+			$settings['button_bg'] = $this->get_button_bg_color( $settings );
+
+			$background_color_obj = new Tribe__Utils__Color( $settings['button_bg'] );
+			$button_bg_rgb = $background_color_obj->getRgb();
+
+			$settings['button_bg_Rgb'] = $button_bg_rgb['R'];
+			$settings['button_bg_rGb'] = $button_bg_rgb['G'];
+			$settings['button_bg_rgB'] = $button_bg_rgb['B'];
```

Comment:
These index names are difficult to discern the difference. Maybe `button_bg_hex_red`, `button_bg_hex_green`, and `button_bg_hex_blue`?